docs/subscription-plans.rst

```diff
@@ -62,6 +74,92 @@ Optimized**. Besides these, we also offer the CrateDB Cloud **Contract**.
   times the storage capacity.
 
 
+.. _subscription-plans-nodes:
+
+Subscription Plan Tiers and Nodes
+=================================
+
+Besides the hardware composition described above, which outlines the ratio of
+the elements CPU, RAM, and storage HD for each subscription plan, each plan has
+two further dimensions: tiers and nodes.
+
+Tiers are simply multiples of each hardware composition, and therefore allow
+linear scaling within a given plan. For example, the Basic tier of the General
+Purpose plan has 3.5 CPUs, 14 GiB of RAM, and 1 TiB of storage. The next tier
+of that plan, the Pro tier, has 7 CPUs, 28 GiB of RAM, and 2 TiB of storage,
+and so forth.
+
+The number of nodes in a cluster can also be adjusted manually for a given
+plan. Basic tiers of a plan allow scaling between 3, 4, and 5 nodes, whereas
+higher tiers allow scaling up to 15. (If you need more nodes than this, please
+`contact us`_.)
+
+The Development plan differs from the others: it has only a single tier and
+allows scaling between 1-3 nodes.
+
+.. WARNING::
+
+    When you have scaled a Development plan cluster above one node, you cannot
+    scale it back down to a single node again.
+
```

Comment:
Might be good to add another warning here saying that "An even-number of nodes can be used for testing purposes, but is not recommended for production workloads due to the possibility of [split-brain](https://en.wikipedia.org/wiki/Split-brain_(computing)) syndrome in case of network problems."